app/views/corporate/extra/_syndication.html.erb

```diff
@@ -3,7 +3,7 @@
 <div class="l-promo-partnerships promo promo--partnerships">
   <div class="l-promo-partnerships__item">
     <%= heading_tag(level: 3, class: 'promo__heading') do %>
-      <%= link_to tool_path('pension-calculator'), class: 'promo__link' do %>
+      <%= link_to t('corporate_tool_directory.popular_tools.tool_one_url'), class: 'promo__link' do %>
```

Comment:
Since there are only 3 allowed in the design, we can can leave it as is I think. But thanks for the suggestion! :)